src/main/java/com/laserfiche/repository/api/clients/impl/model/APIServerException.java

```diff
@@ -2,26 +2,175 @@
+    public String getOperationId() {
+        return operationId;
+    }
+
+    public void setOperationId(String operationId) {
+        this.operationId = operationId;
+    }
+
+    public APIServerException message(String message) {
+        this.message = message;
+        return this;
+    }
+
+    @Schema(description = "")
+    public String getMessage() {
+        return message;
+    }
+
+    public void setMessage(String message) {
+        this.message = message;
+    }
+
+    public APIServerException errorCode(Integer errorCode) {
+        this.errorCode = errorCode;
+        return this;
+    }
+
+    @Schema(description = "")
+    public Integer getErrorCode() {
+        return errorCode;
+    }
+
+    public void setErrorCode(Integer errorCode) {
+        this.errorCode = errorCode;
+    }
+
+    public APIServerException errorClass(String errorClass) {
+        this.errorClass = errorClass;
+        return this;
+    }
+
+    @Schema(description = "")
+    public String getErrorClass() {
+        return errorClass;
+    }
+
+    public void setErrorClass(String errorClass) {
+        this.errorClass = errorClass;
+    }
+
+    public APIServerException statusCode(Integer statusCode) {
+        this.statusCode = statusCode;
+        return this;
+    }
+
+    @Schema(description = "")
+    public Integer getStatusCode() {
+        return statusCode;
+    }
+
+    public void setStatusCode(Integer statusCode) {
+        this.statusCode = statusCode;
+    }
+
+    public APIServerException errorSource(String errorSource) {
+        this.errorSource = errorSource;
+        return this;
+    }
+
+    @Schema(description = "")
+    public String getErrorSource() {
+        return errorSource;
+    }
+
+    public void setErrorSource(String errorSource) {
+        this.errorSource = errorSource;
+    }
+
+    @Override
+    public boolean equals(java.lang.Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        APIServerException apIServerException = (APIServerException) o;
+        return Objects.equals(this.operationId, apIServerException.operationId) && Objects.equals(this.message,
+                apIServerException.message) && Objects.equals(this.errorCode,
+                apIServerException.errorCode) && Objects.equals(this.errorClass,
+                apIServerException.errorClass) && Objects.equals(this.statusCode,
+                apIServerException.statusCode) && Objects.equals(this.errorSource, apIServerException.errorSource);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(operationId, message, errorCode, errorClass, statusCode, errorSource);
+    }
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("class APIServerException {\n");
+        sb
+                .append("    operationId: ")
```

Comment:
minor but I think its better to have toString generate a less verbose representation. currently, the result includes multiple whitespaces, new lines...
I guess a comma separated list of attribute name/values is better.

Reply:
Again, the code is from Swagger Code gen. But yeah, I agree that it could be done better. I think we can probably have a story that eliminates dependency on Swagger Code in the future.

Reply:
Oh, sorry. 
Yeah, I think it's totally ok for now.